reference/configuration/framework.rst

```diff
@@ -1093,8 +1093,61 @@ version_strategy
 
 **type**: ``string`` **default**: ``null``
 
-The service id of the :doc:`asset version strategy </frontend/custom_version_strategy>`
-applied to the assets.
+The service id of the asset version strategy applied to the assets. In addition
+to your own :doc:`custom asset version strategies </frontend/custom_version_strategy>`,
+Symfony defines two services for its :ref:`built-in strategies <component-assets-versioning>`:
+``assets.empty_version_strategy`` and ``assets.static_version_strategy``.
```

Comment:
``assets.static_version_strategy`` is *not* usable directly. It is an abstract service. So you cannot use it directly in ``version_strategy``. If you want to use a static version, use the ``version`` setting (with a non-empty value) instead (which will create a child service configured with this version)